native/Cargo.toml

```
@@ -9,10 +9,11 @@ crate-type = ["dylib"]
 
```

`dylib` is incorrect and was introduced in a previous version of the project template.

```suggestion
crate-type = ["cdylib"]
```

---

It's likely preventing these build flags from being used: https://github.com/neon-bindings/neon/blob/845425b5aa63d5fbaaebd60d102b2250d06c1e3c/crates/neon-build/src/lib.rs#L37-L38